mesh/pmesh.cpp

```diff
@@ -759,36 +834,34 @@ ParMesh::ParMesh(MPI_Comm comm, istream &input, bool refine)
       sedge_ledge.SetSize(num_sedges);
       shared_edges.SetSize(num_sedges);
       group_sedge.SetDims(GetNGroups()-1, num_sedges);
-      v_to_v = new DSTable(NumOfVertices);
-      GetVertexToVertexTable(*v_to_v);
    }
    else
    {
       group_sedge.SetSize(GetNGroups()-1, 0);   // create empty group_sedge
    }
-   // read and set the sizes of sface_lface, group_sface
+   // read and set the sizes of sface_lface, group_{stria,squad}
    if (Dim >= 3)
    {
       skip_comment_lines(input, '#');
-      int num_sfaces;
-      input >> ident >> num_sfaces; // total_shared_faces
-      sface_lface.SetSize(num_sfaces);
-      shared_faces.SetSize(num_sfaces);
-      group_sface.SetDims(GetNGroups()-1, num_sfaces);
-      faces_tbl = GetFacesTable();
+      int num_sface;
+      input >> ident >> num_sface; // total_shared_faces
+      sface_lface.SetSize(num_sface);
+      group_stria.MakeI(GetNGroups()-1);
+      group_squad.MakeI(GetNGroups()-1);
    }
    else
    {
-      group_sface.SetSize(GetNGroups()-1, 0);   // create empty group_sface
+      group_stria.SetSize(GetNGroups()-1, 0);   // create empty group_stria
+      group_squad.SetSize(GetNGroups()-1, 0);   // create empty group_squad
    }
 
    // read, group by group, the contents of group_svert, svert_lvert,
-   // group_sedge, shared_edges, group_sface, shared_faces
-   //
-   // derive the contents of sedge_ledge, sface_lface
-   int svert_counter = 0, sedge_counter = 0, sface_counter = 0;
+   // group_sedge, shared_edges, group_{stria,squad}, shared_{trias,quads}
+   int svert_counter = 0, sedge_counter = 0;
    for (int gr = 1; gr < GetNGroups(); gr++)
    {
+      skip_comment_lines(input, '#');
+#if 0
```

Comment:
Why the `#if 0`?

Reply:
This was the original implementation. I wanted to keep it as a reminder in case some issue comes up as a result of the change.